OfficeUIFabric/Controls/MSLabel.swift

```diff
@@ -14,6 +14,11 @@ open class MSLabel: UILabel {
             update()
         }
     }
+    @objc open var maxPointSize: CGFloat = .greatestFiniteMagnitude {
```

Comment:
`maxPointSize` => `maxFontSize` - since `MSLabel` does not have "point" entity associated with it - this property is applied to its font.